bot.py

```diff
@@ -558,6 +566,54 @@ async def on_message(self, message):
                         'attachments': attachments
                     }}})
 
+        elif message.content.startswith(f'<@!{self.bot.user.id}> ') and message.channel.type == discord.ChannelType.text and not ctx.guild.get_role(config.modRole) in message.author.roles:
```

Comment:
Edge case would cause this not to trigger in case: `<@!1234567890>hi parakarry`

Reply:
Should be fixed in ee3a624